tests.msbuild

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's essentially build.proj except with just the stuff related to testing. I didn't want to name it tests.proj because then there would be two `.proj` files in the root. We can rename it if that doesn't matter.